pages/k8s/install-manual.md

```diff
@@ -204,23 +204,13 @@ juju deploy charmed-kubernetes
 ```
 
 It is also possible to deploy a specific version of the bundle by including the
-revision number. For example, to deploy the **Charmed Kubernetes** bundle for the Kubernetes 1.21
+revision number. For example, to deploy the **Charmed Kubernetes** bundle for the Kubernetes 1.23
 release, you could run:
 
 ```bash
-juju deploy charmed-kubernetes --revision=733
+juju deploy cs:charmed-kubernetes-862
```

Comment:
we want `cs:` here because we're explicitly talking about an old release; CH will use the CS shim to do the right thing when it encounters a `cs:`-prefixed artifact.

Reply:
do we though? because the word on the street is that that is how you get the wrong version. I mean, I agree with you, this is how it should work, so am happy to put this back in

Reply:
well, `--revision=733` doesn't work at all anywhere at any time anymore.  So if you *need* to reference an old bundle, `cs:foo-$rev` is the only way i know to do it.

Reply:
i did confirm this syntax gives the following:
```
$ jst
Model  Controller  Cloud/Region    Version  SLA          Timestamp
862    k8s-crew    vsphere/Boston  2.9.22   unsupported  09:07:47-05:00

App                    Version   Status  Scale  Charm                  Channel   Rev  Exposed  Message
containerd             go1.13.8  active      5  containerd             stable    200  no       Container runtime available
easyrsa                3.0.1     active      1  easyrsa                stable    441  no       Certificate Authority connected.
etcd                   3.4.5     active      3  etcd                   stable    655  no       Healthy with 3 known peers
flannel                0.11.0    active      5  flannel                stable    619  no       Flannel subnet 10.1.5.1/24
kubeapi-load-balancer  1.18.0    active      1  kubeapi-load-balancer  stable    866  yes      Loadbalancer ready.
kubernetes-master      1.23.6    active      2  kubernetes-master      stable   1106  no       Kubernetes master running.
kubernetes-worker      1.23.6    active      3  kubernetes-worker      stable    838  yes      Kubernetes worker running.
...
```

And that's what i expect for a 1.23 bundle.  also confirmed with `733` giving me a `1.21` bundle.

Reply:
ok